source/app-development/interactive/form.rst

```diff
@@ -97,6 +97,20 @@ part of the app developer can look like:
 
 The most commonly used predefined attributes are given as:
 
+cluster
+  This specifies the cluster the job will be submitted to and is an optional field as the cluster can now be specified in the submit.yml.
+
+  This field can also be a list of items as shown below. A dropdown will appear of those options. Glob style matches are also supported - 
+  if there were two clusters named "cluster1" and "cluster2", an input of  "" or "cluster" would include both clusters in the dropdown.
+
+.. code-block:: yaml
```

Comment:
this code block should be indented the same as the text above. So:

```
  This field can also be a list of items as shown below. A dropdown will appear of those options. Glob style matches are also supported - 
  if there were two clusters named "cluster1" and "cluster2", an input of  "" or "cluster" would include both clusters in the dropdown.

  .. code-block:: yaml

     ---
     cluster:
       - "owens"
       - "pitzer"
```

* notice the yaml block starts indented to match the `c` of `code-block`